docs/workflows.rst

```diff
@@ -247,6 +279,8 @@ Susceptibility Distortion Correction (SDC)
     :show-inheritance:
 
 
+.. _bold_reg:
+
```

Comment:
I think that this in line 300 is not true:
```
the gray/white matter boundary (FreeSurfer's ``?h.white`` surfaces).
```

If I'm not wrong, I'd say the pial surfaces are also used. And that is one of the advantages of `bbregister` over `flirt+bbr`.

I know this is outside the scope of the PR, but I think it is worth fixing now that you are updating the docs.

Reply:
bbregister uses the white surface, not pial.

Reply:
Ok, thanks!